docs/user/TerraformAzureAuthentication.md

```diff
@@ -4,15 +4,15 @@ In order to create and destroy Azure resources on your behalf, Terraform needs a
 
 Your Service Principal or Managed Identity in the Azure subscription requires a **"Contributor"** role to create Azure resources. Follow these links for more information on how to create and retrieve the values to configure Terraform access to Azure. 
 
-- [How to create a Service Principal](./AzureHelpTopics.md#service-principal-using-azurecli)
+- [How to create a Service Principal](./AzureHelpTopics.md#service-principal-using-azurecli) | See Using a Service Principal Below
 
 - [How to create a Managed Identity](./AzureHelpTopics.md#how-to-create-a-user-assigned-managed-identity-with-contributor-role) and [how to assign the Managed Identity to a VM](./AzureHelpTopics.md#how-to-assign-a-user-assigned-managed-identity-to-a-vm)
 
 We recommend using [TF_VAR_name](https://www.terraform.io/docs/cli/config/environment-variables.html#tf_var_name) environment variables to pass the authentication information to Terraform. 
 
 ## Using a Service Principal
 
-To authenticate Terraform to access Azure, you will need to set the following four input variables:
+To authenticate Terraform to access Azure, you will need to set the following four input variables (**or run the following script**):
```

Comment:
Move (**or run the following script**) To its own section where you describe the optional script